### PR TITLE
windows: preflights and ssh permissions fix

### PR DESCRIPTION
**Pull request by cfergeau**
This tries to improve some pain points I had while testing on Windows.

### CODE REVIEW

**Comment by cfergeau** [comment]
Needs more work/testing:
```
panic: Should not happen, empty description for fix 'check-administrator-user'

goroutine 1 [running]:
github.com/crc-org/crc/v2/pkg/crc/preflight.(*Check).doFix(0xc00079fa10?)
        /home/teuf/dev/crc/pkg/crc/preflight/preflight.go:72 +0x119
github.com/crc-org/crc/v2/pkg/crc/preflight.doFixPreflightChecks({0x2cb33b0, 0xc00021f770}, {0xc000489000?, 0x17, 0xc0004f12d9?}, 0x0)
        /home/teuf/dev/crc/pkg/crc/preflight/preflight.go:116 +0x110
github.com/crc-org/crc/v2/pkg/crc/preflight.SetupHost({0x2cb33b0, 0xc00021f770}, 0xb4?)
        /home/teuf/dev/crc/pkg/crc/preflight/preflight.go:174 +0x45
github.com/crc-org/crc/v2/cmd/crc/cmd.runSetup({0x22bc24014c?, 0xc00079fbb0?, 0xb49b97?})
        /home/teuf/dev/crc/cmd/crc/cmd/setup.go:69 +0x29c
github.com/crc-org/crc/v2/cmd/crc/cmd.glob..func10(0xc00013c000?, {0x4419348, 0x0, 0x0})
        /home/teuf/dev/crc/cmd/crc/cmd/setup.go:42 +0x5d
github.com/crc-org/crc/v2/cmd/crc/cmd.executeWithLogging.func1(0x3e701a0, {0x4419348, 0x0, 0x0})
        /home/teuf/dev/crc/cmd/crc/cmd/root.go:169 +0x102
github.com/spf13/cobra.(*Command).execute(0x3e701a0, {0x4419348, 0x0, 0x0})
        /home/teuf/dev/crc/vendor/github.com/spf13/cobra/command.go:983 +0xaaa
github.com/spf13/cobra.(*Command).ExecuteC(0x3e6fec0)
        /home/teuf/dev/crc/vendor/github.com/spf13/cobra/command.go:1115 +0x425
github.com/spf13/cobra.(*Command).Execute(...)
        /home/teuf/dev/crc/vendor/github.com/spf13/cobra/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /home/teuf/dev/crc/vendor/github.com/spf13/cobra/command.go:1032
github.com/crc-org/crc/v2/cmd/crc/cmd.Execute()
        /home/teuf/dev/crc/cmd/crc/cmd/root.go:105 +0xd9
main.main()
        /home/teuf/dev/crc/cmd/crc/main.go:8 +0x17
```

/hold

**Comment by cfergeau** [comment]
> Needs more work/testing:

Fixed now.

File: pkg/crc/preflight/preflight_windows.go [hunk 1]
```diff
@@ -19,39 +19,44 @@ var hypervPreflightChecks = []Check{
 		configKeySuffix:  "check-administrator-user",
 		checkDescription: "Checking if running in a shell with administrator rights",
 		check:            checkIfRunningAsNormalUser,
-		flags:            StartUpOnly,
+		fixDescription:   "crc should be ran in a shell without administrator rights",
```
Comment (gbraad) [inline pkg/crc/preflight/preflight_windows.go]: `should be ran`?

Why the passive voice wth a past tense?

**Review by gbraad — COMMENTED** [review]
Can you address the title as follows: `Preflights and ssh permissions fix.` ?

**Review by gbraad — CHANGES_REQUESTED** [review]
This should not be with the past simple form for passive voice (past participle instead) and description just states 'improvements' while it is more clear when 'permissions' is mentioned.

File: pkg/crc/preflight/preflight_windows.go (hunk 1 above)
Comment (cfergeau) [reply pkg/crc/preflight/preflight_windows.go]: This used to be this way, I only c&p'ed what was removed in https://github.com/crc-org/crc/commit/1ef7f7b5ffc837835dfc0fe4a0865e8ddf2a164c
I'll change it. This should be "should be run", right?

File: pkg/crc/preflight/preflight_checks_windows.go [hunk 2]
```diff
@@ -184,7 +184,7 @@ func checkIfRunningAsNormalUser() error {
 		return nil
 	}
 	logging.Debug("Ran as administrator")
-	return fmt.Errorf("crc should be ran in a shell without administrator rights")
+	return fmt.Errorf("crc should be run in a shell without administrator rights")
```
Comment (gbraad) [inline pkg/crc/preflight/preflight_checks_windows.go:187]: 👍 (past participle for passive voice!)

**Comment by gbraad** [comment]
> This tries to improve some pain points I had while testing on Windows.

Can you reference in future the issue this tries to solve?

**Review by gbraad — APPROVED** [review]

**Comment by cfergeau** [comment]
> > This tries to improve some pain points I had while testing on Windows.
> 
> Can you reference in future the issue this tries to solve?

I've added "fixes https://github.com/crc-org/crc/issues/4079" to the 2nd commit log.

**Comment by gbraad** [comment]
OKD test now fails?

**Comment by cfergeau** [comment]
> OKD test now fails?

Looks like a timeout of some sort.

**Review by anjannath — APPROVED** [review]

**Pull request merged**